asm/macros/event.inc

```diff
@@ -251,9 +251,18 @@
 	.endm
 
 	@ Calls the native C function stored at func.
-	.macro callnative func:req
+	@ mutating should be set to FALSE if the native cannot have a
+	@ mutating effect in the sense documented in Script_HasNoEffect.
+	@ If the native sometimes has a mutating effect when called with
+	@ its arguments, it must be treated as always having a mutating
+	@ effect and set to TRUE.
+	.macro callnative func:req, mutating=TRUE
```

Comment:
I would add a comment that adding `ROM_SIZE` mirrors gba memory and `func` is still valid and will run normally after.
This could look scary if someone doesn't know what it does.